tests/drivers/spi/spi_loopback/boards/teensy41.conf

```diff
@@ -0,0 +1,7 @@
+#
```

Comment:
There is no support for teensy41 upstream in Zephyr so adding this file doesn't make any sense.